src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java

```diff
@@ -162,26 +193,33 @@ private ReadFilterLibrary(){ /*no instance*/ }
             return read.isUnmapped() ||
                     read.getLength() == Cigar.getReadLength(read.getCigarElements());}}
 
+    /** Keep only reads that are second of pair. */
     @DocumentedFeature(groupName=HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
     public static class SecondOfPairReadFilter extends ReadFilter {
         private static final long serialVersionUID = 1L;
         @Override public boolean test (final GATKRead read) {
             return read.isSecondOfPair();}}
 
+    /** Keep reads with sequenced bases. */
     @DocumentedFeature(groupName=HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
     public static class SeqIsStoredReadFilter extends ReadFilter {
         private static final long serialVersionUID = 1L;
         @Override public boolean test(final GATKRead read){
             return read.getLength() > 0;}}
 
+    /** Keep reads with a valid alignment start (larger than 0, or unmapped).*/
```

Comment:
Added clarification and extraDocs to have the url. I also search for isUnmapped usages in the filter library and added the last line.